public/js/searchList.js

```diff
@@ -0,0 +1,79 @@
+let searchCardsTemplate;
+
+/*
+ * When the page loads, fetch initial organization data and render it
+ * in cards on the list.
+ */
+window.onload = function() {
+  fetchOrganizations("all").then(organizations => {
+    createOrganizationCards(organizations);
+    createMarkers(organizations);
+  })
+}
+
+/*
+ * Fetches the organization data with the given filters from the server.
+ * @param {string} filterText The selected item category with which to filer results for.
+ */
+function fetchOrganizations(filterText) {
+  return fetch("/discover", {
+    method: 'POST',
+    body: JSON.stringify({filter: filterText})
+  }).then(data => data.json());
+}
+
+/*
+ * Renders the organization data into cards.
+ * @param {JSON object} organizationsToRender The JSON of organization data to add to the page.
+ */
+function createOrganizationCards(organizationsToRender) {
+  let cardList = document.getElementById("search-list");
+  let template = document.getElementsByTagName("template")[0].content;
```

Comment:
Is there a way to add an ID attribute `<template>` so then you can refer to it using `getElementById` instead of having to use `getElementsByTagName` and dealing with a list?